test/test_grocy.py

```diff
@@ -218,6 +218,16 @@ def test_consume_product_error(self):
         )
         self.assertRaises(GrocyError, self.grocy.consume_product, 1, 1.3)
 
+    @responses.activate
```

Comment:
I'm trying to get rid of responses and switch to cassettes instead like in most other tests. Could you migrate to these please?